docs/guide/webmcp-webskills.md

```
@@ -243,18 +248,35 @@ const skillMdModules = import.meta.glob('./skills/**/*', {
+        { action: 'remote-control', text: '识别码', desc: result.sessionId.slice(-6), know: true, showCopyIcon: true }
+      ]
+    }
+  } catch (err) {
+    console.warn('[WebAgent] 远程遥控初始化失败，本地功能不受影响：', err)
+  }
 })
 </script>
 ```
 
-> 中文小结：通过 `skills + mcpServers`，TinyRemoter 就能在一个对话面板里同时掌握“业务知识 + 业务工具”，真正做到“对话即页面操作”。
+> 中文小结：`menuItems` 在 WebAgent 连接成功后填充，TinyRemoter 会自动在悬浮菜单中显示"遥控器链接"和"识别码"。本地 MCP 与远程遥控**必须分开 try/catch 处理**，避免网络问题导致本地对话功能也一起失效。详细接入方式见 [远程遥控亮点章节](#24-远程遥控跨设备遥控桌面的杀手级亮点)。
```

markdownlint-cli2 (0.21.0)

[warning] Link fragments should be valid

(MD051, link-fragments)